open-machine-learning-jupyter-book/data-science/working-with-data/Indexing_on_ndarrays.ipynb

```diff
@@ -0,0 +1,2166 @@
+{
```

Comment:
<p>A secondary heading cannot be used as the first title of a document.</p><p>The solution is to add a first-level heading in front of it. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/608/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>